builtin/providers/scaleway/data_source_scaleway_bootscript_test.go

```diff
@@ -2,25 +2,44 @@ package scaleway
 
 import (
 	"fmt"
+	"os"
 	"regexp"
 	"testing"
 
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/terraform"
+	"github.com/scaleway/scaleway-cli/pkg/api"
+	"github.com/scaleway/scaleway-cli/pkg/scwversion"
 )
 
 func TestAccScalewayDataSourceBootscript_Basic(t *testing.T) {
+	testAccPreCheck(t)
+	client, err := api.NewScalewayAPI(
```

Comment:
@stack72 I tried `testAccProvider.Meta().(*Client).scaleway` but Meta returned nil. it was only present if a) I have a test with multiple steps and read it after the first `TestStep` or b) use it inside the verify function.

Sadly I did not find the place where `SetMeta` is being called, so I'm not sure why it's not available there.